index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+    <nav id="navbar">
+        <div id="container-logo"><a id="myLogo" >My Logo</a></div>     
+        <div id="container-menu"><a id="menu"><img src="css/img/hamburger.svg"></a></div> 
+    </nav>

```

Comment:
- [ ] I would kindly suggest this part of the page has the container as an `header`.
   - The container of the section is a `header` tag.
        - Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

- [ ] Great effort wrapping the text "My Logo" inside of an anchor `a` tag, but I would kindly advice you move the logo outside the `nav` tag but inside of the `header` tag
     - Because The logo (if the design has one) is on the left and has a link (a tag).
          - This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

- [ ] Lastly the hamburger button should be wrap inside of a `nav` tag and should be on the right hand side.
   - The hamburger button is on the right, and has a `nav` tag.
      - The `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices.

You can simply restructure your line of code to something like this :point_down: 

```HTML
<header>
   <div id="container-logo"><a id="myLogo" >My Logo</a></div>
   <nav id="navbar">     
        <div id="container-menu"><a id="menu"><img src="css/img/hamburger.svg"></a></div> 
    </nav>
</header>
```